integration-tests/src/test/resources/bash-scripts/images.properties

```diff
@@ -0,0 +1,60 @@
+
+container-registry.oracle.com/middleware/weblogic:14.1.1.0-slim-8;weblogick8s/test-images/weblogic:14.1.1.0-slim-8
+container-registry.oracle.com/middleware/weblogic:14.1.1.0-slim-8-ol8;weblogick8s/test-images/weblogic:14.1.1.0-slim-8-ol8
+
+container-registry.oracle.com/middleware/weblogic:14.1.1.0-dev-11;weblogick8s/test-images/weblogic:14.1.1.0-dev-11
+container-registry.oracle.com/middleware/weblogic:14.1.1.0-dev-11-ol8;weblogick8s/test-images/weblogic:14.1.1.0-dev-11-ol8
+
+container-registry.oracle.com/middleware/weblogic:14.1.1.0-dev-8;weblogick8s/test-images/weblogic:14.1.1.0-dev-8
+container-registry.oracle.com/middleware/weblogic:14.1.1.0-dev-8-ol8;weblogick8s/test-images/weblogic:14.1.1.0-dev-8-ol8
+
+# WebLogic Release 12.2.1.4
+
+container-registry.oracle.com/middleware/weblogic:12.2.1.4;weblogick8s/test-images/weblogic:12.2.1.4
+container-registry.oracle.com/middleware/weblogic:12.2.1.4-ol8;weblogick8s/test-images/weblogic:12.2.1.4-ol8
+
+container-registry.oracle.com/middleware/weblogic:12.2.1.4-slim;weblogick8s/test-images/weblogic:12.2.1.4-slim
+container-registry.oracle.com/middleware/weblogic:12.2.1.4-slim-ol8;weblogick8s/test-images/weblogic:12.2.1.4-slim-ol8
+
+container-registry.oracle.com/middleware/weblogic:12.2.1.4-dev;weblogick8s/test-images/weblogic:12.2.1.4-dev
+container-registry.oracle.com/middleware/weblogic:12.2.1.4-dev-ol8;weblogick8s/test-images/weblogic:12.2.1.4-dev-ol8
+
+# WebLogic Release 12.2.1.3
+
+container-registry.oracle.com/middleware/weblogic:12.2.1.3;weblogick8s/test-images/weblogic:12.2.1.3
+container-registry.oracle.com/middleware/weblogic:12.2.1.3-ol8;weblogick8s/test-images/weblogic:12.2.1.3-ol8
+
+container-registry.oracle.com/middleware/weblogic:12.2.1.3-dev;weblogick8s/test-images/weblogic:12.2.1.3-dev
+container-registry.oracle.com/middleware/weblogic:12.2.1.3-dev-ol8;weblogick8s/test-images/weblogic:12.2.1.3-dev-ol8
+
+# FMW Release 12.2.1.4
+
+container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4;weblogick8s/test-images/fmw-infrastructure:12.2.1.4
+container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4-ol8;weblogick8s/test-images/fmw-infrastructure:12.2.1.4-ol8
+
+# FMW Release 12.2.1.3
+container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3;weblogick8s/test-images/fmw-infrastructure:12.2.1.3
+container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3-ol8;weblogick8s/test-images/fmw-infrastructure:12.2.1.3-ol8
+
+# Oracle DB Image Release 12.2.0.1-slim
+container-registry.oracle.com/database/enterprise:12.2.0.1-slim;weblogick8s/test-images/database/enterprise:12.2.0.1-slim
+
+# BusyBox
+busybox:1.34.1;weblogick8s/test-images/docker/busybox:1.34.1
+
+# NGINX Controller
+k8s.gcr.io/ingress-nginx/controller:v1.2.0;weblogick8s/test-images/ingress-nginx/controller:v1.2.0
+
+# Oracle DB Operator Image
+container-registry.oracle.com/database/operator:0.1.0;weblogick8s/test-images/database/operator:0.1.0
```

Comment:
Add another line to this file 
elasticsearch:7.8.1;weblogick8s/test-images/docker/elasticsearch:7.8.1